packages/constants/index.ts

```diff
@@ -14,6 +14,14 @@ export const PERSISTENCE_CACHE_TIME = 1000 * 60 * 60 * 12; // 12 hours
 export const BTC_DECIMALS = 8;
 export const STX_DECIMALS = 6;
 
+// Units of `Money` should be declared in their smallest unit. Similar to
```

Comment:
Is this necessary in this PR?

Reply:
It'd be nicer for this to live with money, but think it's okay to move here, as these are constant values.